xsl/pretext-html.xsl

```diff
@@ -3807,6 +3839,26 @@ along with MathBook XML.  If not, see <http://www.gnu.org/licenses/>.
                         </xsl:apply-templates>
                     </xsl:if>
                 </xsl:when>
+                <!-- webwork with stages             -->
+                <xsl:when test="webwork-reps/static/stage">
+                    <xsl:apply-templates select="webwork-reps/static/stage" mode="exercise-components">
+                        <xsl:with-param name="b-original" select="false()" />
+                        <xsl:with-param name="b-has-statement" select="$b-has-statement" />
+                        <xsl:with-param name="b-has-hint"      select="$b-has-hint" />
+                        <xsl:with-param name="b-has-answer"    select="$b-has-answer" />
+                        <xsl:with-param name="b-has-solution"  select="$b-has-solution" />
+                    </xsl:apply-templates>
+                </xsl:when>
+                <!-- webwork without stages          -->
+                <xsl:when test="webwork-reps">
```

Comment:
This bit with `test="webwork-reps"` was cut along with the `stage` stuff. If you leave this part in, it still won't do good things for when there is a `stage`, but it will work the other 99.9% of the time.

Reply:
I'm happy to open a new PR that puts this back in. Before doing that, I wanted to understand if you believe that is the right thing to do (adding back in this chunk). Or if you would prefer to wait for stage to have its refactor at the same time.